formula1-dao/src/test/java/cz/muni/pa165/teamwhite/formula1/persistence/DriverTest.java

```diff
@@ -123,40 +93,24 @@ public void createDriverWithCarTest() {
 
         Driver dbDriver = driverDao.findById(driver.getId());
 
-        Assert.assertNotNull(dbDriver);
-        Assert.assertNotNull(dbDriver.getId());
-        Assert.assertNotEquals(dbDriver.getId(), 0L);
-        Assert.assertEquals(dbDriver.getName(), driver.getName());
-        Assert.assertEquals(dbDriver.getSurname(), driver.getSurname());
-        Assert.assertEquals(dbDriver.getNationality(), driver.getNationality());
-        Assert.assertEquals(dbDriver.isAggressive(), driver.isAggressive());
-        Assert.assertEquals(dbDriver.getWetDriving(), driver.getWetDriving());
-        Assert.assertEquals(dbDriver.getReactions(), driver.getReactions());
         Assert.assertEquals(dbDriver.getCar().getName(), driver.getCar().getName());
         Assert.assertEquals(dbDriver.getCar().getName(), car.getName());
     }
 
     @Test
-    public void findAllDriversTest() {
-        Driver driver = new Driver();
-        driver.setName("Lewis");
-        driver.setSurname("Hamilton");
-        driver.setNationality("GB");
-        driver.setIsAggressive(true);
-        driver.setWetDriving(10);
-        driver.setReactions(10);
-
+    public void tryToCreateTwoIdenticalDriverTest() {
         driverDao.create(driver);
+        Assert.assertEquals(driverDao.findAll().size(), 1);
 
+        driverDao.create(driver);
         Assert.assertEquals(driverDao.findAll().size(), 1);
```

Comment:
```suggestion
         Driver driver1 = new Driver(null, "Lewis", "Hamilton", "GB", true, 10, 10);
        Driver driver2 = new Driver(null, "Lewis", "Hamilton", "GB", true, 10, 10);

        driverDao.create(driver1);
        Assert.assertEquals(driverDao.findAll().size(), 1);

        driverDao.create(driver2);
        Assert.assertEquals(driverDao.findAll().size(), 2);
```


Pokud by jsme chteli testovat ulozeni 2 identickych ridicu, take by jsme to mueli udelat nejak takto. Kdyz 2x ulozis stejnou entitu, tak se nic nestane.

[`if an instance is already persistent, then this call has no effect for this particular instance (but it still cascades to its relations with cascade=PERSIST or cascade=ALL),`](https://www.baeldung.com/hibernate-save-persist-update-merge-saveorupdate#1-persist)